app/helpers/service_helper.rb

```diff
@@ -19,6 +19,10 @@ def tools_url
     service_url_for('/tools')
   end
 
+  def previous_service_url(commodity_code)
```

Comment:
with previous, we mean the service who called trade tariff right?
that's not super clear, 
you have a better name? maybe `caller_service_url` ...

Reply:
The caller to me is the person making the current request, not the previous service they arrived from